scripts/do-release.sh

```diff
@@ -35,15 +35,19 @@ function create_project_release() {
         return
     fi
 
+    # If the project has container images, copy them to the release tag
+    # This will fail if the source images don't exist; abort then without trying to create the release
+    if [[ -n "$(project_images)" ]]; then
+        if ! tag_images "$(project_images)"; then
+            ((errors++))
+            return 1
+        fi
+    fi
```

Comment:
```suggestion
    if [[ -n "$(project_images)" ]] && ! tag_images "$(project_images)"; then
        ((errors++))
        return 1
    fi
```

I think this makes it simpler to read?